tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/jvm/CodegenErrorsTest.java

```diff
@@ -73,4 +73,10 @@ public void testLargeMethods() {
         CompileResult result = BCompileUtil.compile("test-src/jvm/largeMethods");
         BRunUtil.invoke(result, "main");
     }
+
+    @Test
+    public void testLargeNumberOfListeners() {
+        CompileResult result = BCompileUtil.compile("test-src/jvm/large-number-of-listeners.bal");
+        Assert.assertEquals(result.getErrorCount(), 0);
+    }
```

Comment:
Shall we also run this code as well during test. We can increment global count during start and check whether the count is == listener count during main fucntion

Reply:
Now runs the code during the test.